ngraph/test/runtime/interpreter/unit_test.manifest

```diff
@@ -29,6 +29,11 @@ INTERPRETER.onnx_model_matmul_integer_scalar
 INTERPRETER.onnx_model_matmul_integer_4d
 INTERPRETER.onnx_model_matmul_integer_4d_zero_point
 INTERPRETER.onnx_model_matmul_integer_4d_no_zero_point
+INTERPRETER.deformable_convolution_2D_1batch_1channel
+INTERPRETER.deformable_convolution_2D_1batch_1channel_stride
+INTERPRETER.deformable_convolution_2D_1batch_1channel_dilation
+INTERPRETER.deformable_convolution_2D_1batch_2filter
+INTERPRETER.deformable_convolution_2D_2batch_1channel
```

Comment:
Why these are ignored ? From my understanding tests with offsets == 0 should work, right?

Reply:
They all have offsets ! = 0

Reply:
Okej then ... maybe if the test name include this information it would be clearer for the reader :)